internals-js/src/definitions.ts

```diff
@@ -3201,13 +3201,48 @@ function *typesToCopy(source: Schema, dest: Schema): Generator<NamedType, void,
   yield* source.types();
 }
 
+/**
+ * Iterate through schema directives in a safe order. This means a directive
+ * that's used by another directive will be yielded first.
+ *
+ * For example, the directive order in this schema will be `@bar` then `@foo`
+ * because `@foo` uses `@bar`.
+ *
+ * ```graphql
+ * directive @foo(arg: String @bar) on FIELD_DEFINITION
+ * directive @bar on ARGUMENT_DEFINITION
+ * ```
+ */
+function *walkDirectives(schema: Schema): Generator<DirectiveDefinition, void, undefined> {
```

Comment:
I don't love this method here, but wasn't sure of a better place for it. Does it belong as a method on `Schema`?

Reply:
Even better, this method is totally unnecessary